node/typings.d.ts

```diff
@@ -156,8 +160,20 @@ interface Price {
   id: string
 }
 
+interface CustomField {
+  name: string
+  type: 'text'
```

Comment:
This interface is identical to `CustomFieldSetting`, I think it is supposed to have a value too?

Reply:
✅ CustomField was missing a value